DiscordChatExporter.Gui/Views/Dialogs/ExportSetupView.xaml.cs

```diff
@@ -1,4 +1,8 @@
-﻿namespace DiscordChatExporter.Gui.Views.Dialogs
+﻿using DiscordChatExporter.Gui.Internal;
+using System;
+using System.Windows;
+
+namespace DiscordChatExporter.Gui.Views.Dialogs
```

Comment:
```suggestion
﻿namespace DiscordChatExporter.Gui.Views.Dialogs
```

Looks like these were added with some other changes, but are no longer needed